arm-logic/2016-06-01/examples/SessionsGet.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "Get integration account session examples",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123",
+    "sessionName": "testsession123-ICN"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Cache-Control": "no-cache",
+        "Pragma": "no-cache",
+        "Content-Length": 515,
+        "Content-Type": "application/json; charset=utf-8",
+        "Expires": -1,
+        "ETag": "08587138971894986508",
+        "Vary": "Accept-Encoding",
+        "Date": "Tue, 21 Feb 2017 20:43:03 GMT"
+      },
+      "body": {
+        "properties": {
+          "createdTime":"2017-02-21T20:41:35.9789189Z",
+          "changedTime":"2017-02-21T20:41:35.9796534Z",
+          "content":"{\"controlNumber\":\"1000\", \"controlNumberChangedTime\":\"2017-02-15T00:36:00.9140158Z\"}"
+        },
+        "id":"/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Logic/integrationAccounts/testia123/sessions/testsession123-ICN",
+        "name":"testsession123-ICN",
+        "type":"Microsoft.Logic/integrationAccounts/sessions",
+        "etag":"\"08587138971894986508\""
```

Comment:
"etag" is not defined in the swagger spec https://travis-ci.org/Azure/azure-rest-api-specs/jobs/204011630#L2367